commit 921f15fe8c8cf9543665bdc556e080a5ecbc34b3
Author: Kees Cook <kees@kernel.org>
Date:   Mon Sep 25 10:17:36 2023 -0700

    MAINTAINERS: hardening: Add __counted_by regex
    
    Since __counted_by annotations may also require that code be changed to
    get initialization ordering correct, let's get an extra group of eyes on
    code that is working on these annotations.
    
    Signed-off-by: Kees Cook <keescook@chromium.org>

diff --git a/MAINTAINERS b/MAINTAINERS
index 737dcc7a2155..741285b8246e 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -11405,6 +11405,7 @@ F:	kernel/configs/hardening.config
 F:	mm/usercopy.c
 K:	\b(add|choose)_random_kstack_offset\b
 K:	\b__check_(object_size|heap_object)\b
+K:	\b__counted_by\b
 
 KERNEL JANITORS
 L:	kernel-janitors@vger.kernel.org